io/driver/driver_parameter.py.data/driver_parameter_mlx4_core.yaml

```diff
@@ -0,0 +1,160 @@
+module: "mlx4_core"
+interface: eth3
+host_ip:
+netmask:
+peer_ip:
+sysfs_check_required: 1
+value: !mux
+    mlx4_en_only_mode_0:
+       module_param_name: "mlx4_en_only_mode"
+       module_param_value: "0"
+    mlx4_en_only_mode_1:
+       module_param_name: "mlx4_en_only_mode"
+       module_param_value: "1"
```

Comment:
@manvanthar Can you make this similar to https://github.com/avocado-framework-tests/avocado-misc-tests/blob/67ea45171364506869cdce12423559e211aefd07/io/driver/driver_parameter.py.data/driver_parameter_mlx5_ib.yaml

Reply:
And all other parameters in all files as well